vdr/src/client/client.rs

```diff
@@ -213,10 +213,28 @@ impl Debug for LedgerClient {
 #[cfg(test)]
 pub mod test {
     use super::*;
-    use crate::types::EventLog;
+    use crate::{
+        client::MockClient, signer::basic_signer::test::basic_signer, types::EventLog, Role,
+    };
     use async_trait::async_trait;
     use once_cell::sync::Lazy;
-    use std::{env, fs};
+    use std::{env, fs, sync::RwLock};
+
+    pub const CONTRACT_NAME_EXAMPLE: &str = "ValidatorControl";
+    pub const CONTRACT_METHOD_EXAMPLE: &str = "addValidator";
+    pub const VALIDATOR_LIST_BYTES: Lazy<Vec<u8>> = Lazy::new(|| {
```

Comment:
Why are these constants are defined here but not in the corresponding contract file?

Reply:
Moved